<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSTimesManager</class>
 <widget class="QWidget" name="RSTimesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>91</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RSTimesManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="m_border">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_title">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="m_body">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Start Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="m_startDateEdit">
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>End Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="m_endDateEdit">
             <property name="calendarPopup">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Step Duration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_stepDateEdit"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="m_stepViewLabel">
             <property name="text">
              <string>Step Max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_stepViewEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>10000000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
